### PR TITLE
Feature/structure

### PR DESCRIPTION
**Pull request by matthijs186**
**TEST EXTENSIVELY!** We don't want any missing references or stuff at the wrong place.

### CODE REVIEW

File: src/assets/scss/style.scss [hunk 1]
```diff
@@ -1,11 +1,11 @@
 // Vendor
-@import 'vendor/_normalize.scss';
```
Comment (ferrybig) [inline src/assets/scss/style.scss]: If you decide to change the quotes, make it consistent in the whole file, instead of only a few lines

**Review by Drelofs — APPROVED** [review]
👍

**Review by ferrybig — APPROVED** [review]

**Review by RishiTheGoat — APPROVED** [review]
🥇

**Review by peterxqe — APPROVED** [review]
works for me m8

**Pull request merged**